react-frontend/src/components/listRecognitions.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Row, Col, Label, Button, Container } from "reactstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { faEllipsisV } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  MenuButton,
+  MenuList,
+  MenuItem,
+  MenuLink,
+  Menu,
+} from "@reach/menu-button";
+import "@reach/menu-button/styles.css";
+const ListRecognition = () => {
+  const persons = [
+    {
+      hi5: "1",
+      name: "AM",
+      time: "1 min ago",
+      text:
+        "recognition text:welcome to bonusly @sahil+bonasuly ! we are recognistion you with +25",
+      core: "welcome to bonusly",
+    },
+    {
+      hi5: "2",
+      name: "JB",
+      time: "2 min ago",
+      text:
+        "recognition text:welcome to bonusly @sahil+bonasuly ! we are recognistion you with +25",
+      core: "welcome to bonusly",
+    },
+    {
+      hi5: "3",
+      name: "OH",
+      time: "3 min ago",
+      text:
+        "recognition text:welcome to bonusly @sahil+bonasuly ! we are recognistion you with +25",
+      core: "welcome to bonusly",
+    },
+  ];
+  return (
+    <Container>
+      <Row>
+        <Col xs="3"></Col>
+        <Col>
+          {persons.map((person) => (
```

Comment:
Do not use `inline functions` in render. Move the code inside this loop to a separate `Person` component.